app/styles/layout/_select-inline.scss

```diff
@@ -204,6 +204,7 @@ $list-height: 220px;
       margin: 5px 7px 0 0;
       position: absolute;
       width: $icon-width;
+      left: 10px;
```

Comment:
@filipecrosk I would maybe move this above `margin` for alphabetical ordering and then just comment that this is a fix for Firefox inline, so we know. Can you show a screenshot of the fix, btw?